### PR TITLE
Dilpreet | Test | Even when expiry date is coming from API then also expired is showing in download page

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2me3k6j

### CODE REVIEW

File: apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts [hunk 1]
```diff
@@ -128,9 +128,11 @@ export class DownloadsComponent implements OnInit, OnDestroy {
             if (response && response.status === 'success') {
                 response.body?.items?.forEach((result: any) => {
                     result.date = moment(result.date, GIDDH_DATE_FORMAT + " HH:mm:ss").format(GIDDH_DATE_FORMAT);
-                    let today = moment().format(GIDDH_DATE_FORMAT);
-                    if (result.expireAt >= today) {
-                        result.expireAt = moment(result.expireAt, GIDDH_DATE_FORMAT + " HH:mm:ss").format(GIDDH_DATE_FORMAT);
+                    let todayReverse = moment().format('YYYY-MM-DD');
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts]:
```suggestion
                    let today = moment().format('YYYY-MM-DD');
```

File: apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts [hunk 2]
```diff
@@ -128,9 +128,11 @@ export class DownloadsComponent implements OnInit, OnDestroy {
...
+                    let expiredReverse = moment(result.expireAt, GIDDH_DATE_FORMAT + " HH:mm:ss").format('YYYY-MM-DD');
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts]:
```suggestion
                    let expired = moment(result.expireAt, GIDDH_DATE_FORMAT + " HH:mm:ss").format('YYYY-MM-DD');
```

File: apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts [hunk 3]
```diff
@@ -128,9 +128,11 @@ export class DownloadsComponent implements OnInit, OnDestroy {
...
+                    if (moment(expiredReverse)
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts]:
```suggestion
                    if (moment(expired)
```

File: apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts [hunk 4]
```diff
@@ -128,9 +128,11 @@ export class DownloadsComponent implements OnInit, OnDestroy {
...
+                        .isSameOrAfter(todayReverse)) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts]:
```suggestion
                        .isSameOrAfter(today)) {
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts [hunk 5]
```diff
@@ -128,9 +128,11 @@ export class DownloadsComponent implements OnInit, OnDestroy {
...
+                        result.expireAt = result.expireAt = moment(result.expireAt, GIDDH_DATE_FORMAT + " HH:mm:ss").format(GIDDH_DATE_FORMAT);
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/downloads/components/downloads/downloads.component.ts]:
```suggestion
                        result.expireAt = moment(result.expireAt, GIDDH_DATE_FORMAT + " HH:mm:ss").format(GIDDH_DATE_FORMAT);
```

**Review by the-prateek-mishra — APPROVED** [review]

**Comment by dilpreetsinghofficial** [comment]
PR reviewed

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**